view/theme/frio/templates/sub/direction.tpl

```diff
@@ -0,0 +1,10 @@
+{{if $direction.direction > 0}}
+<span class="direction">
+	&bull;
+	{{if $direction.direction == 1}}
+		<i class="fa fa-inbox" aria-hidden="true" title="{{$direction.title}}"></i>
+	{{elseif $direction.direction == 2}}
+		<i class="fa fa-download" aria-hidden="true" title="{{$direction.title}}"></i>
+	{{/if}}
```

Comment:
This won't show anything if the direction is unknown, is it intended?

Reply:
Yes. We currently only set this value with AP posts, since there the situation can happen that a starting post is fetched because a subscribed contact posted a comment in that thread. With this indicator we can now detect those situations.